test cases/common/263 response file/meson.build

```diff
@@ -0,0 +1,37 @@
+project('response file test')
+
+
+
+
+
+args = ['a', 'b', 'c']
+
+rf = response_file(
+    args, 'd', 'e', 'f',
+    max_args_length: false,
+)
+assert(rf[0] == 'a')  # firt item of the list
+assert(rf[5] == 'f')  # list is passed as-is
+
+rf = response_file(
+    args, 'd', 'e', 'f',
+    max_args_length: true,
+)
+assert(rf[0] == '@')
+
+
+pymod = import('python')
+py = pymod.find_installation()
+
+rf = response_file(
```

Comment:
How do users differentiate between the two return types? Feels kind of hard...

Reply:
See my other comment above. Since the result is always meant to be used in the argument list of a command, it souldn't really matter whether the value is a string or a list.

Reply:
I think we should always return a list, Meson is flatten all lists anyway.